test/groovy/InfluxWriteDataTest.groovy

```diff
@@ -0,0 +1,97 @@
+#!groovy
+import com.lesfurets.jenkins.unit.BasePipelineTest
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import util.JenkinsSetupRule
+
+import static com.lesfurets.jenkins.unit.MethodSignature.method
+import static org.junit.Assert.assertTrue
+import static org.junit.Assert.assertEquals
+
+class InfluxWriteDataTest extends BasePipelineTest {
+
+    Map fileMap = [:]
+    Map stepMap = [:]
+    String echoLog = ''
+
+    @Rule
+    public JenkinsSetupRule setupRule = new JenkinsSetupRule(this)
+
+    @Before
+    void setUp() throws Exception {
+        super.setUp()
+
+        helper.registerAllowedMethod('readYaml', [Map.class], { map ->
+            return [
+                general: [productiveBranch: 'develop'],
+                steps : [influxWriteData: [influxServer: 'testInflux']]
+            ]
+        })
+
+        helper.registerAllowedMethod('writeFile', [Map.class],{m -> fileMap[m.file] = m.text})
+        helper.registerAllowedMethod('step', [Map.class],{m -> stepMap = m})
+        helper.registerAllowedMethod("echo", [String.class], {s -> echoLog += s})
+    }
+
+
+    @Test
+    void testInfluxWriteDataWithDefault() throws Exception {
+
+        def cpe = loadScript('commonPipelineEnvironment.groovy').commonPipelineEnvironment
+        cpe.setArtifactVersion('1.2.3')
+        def script = loadScript("influxWriteData.groovy")
```

Comment:
Proposal: should we name the script under tests in a more remarkable way in order to increase the recognizability. My suggestion would be ```examinee```.

Reply:
I suggest to name it like the step itself with `Script` appended.

This will make it crystal clear.

In this case it will be `influxWriteDataScript`